qa/tasks/s3tests.py

```diff
@@ -726,6 +647,7 @@ def task(ctx, config):
                         's3 main'    : {},
                         's3 alt'     : {},
                         's3 tenant'  : {},
+                        'iam'        : {},
```

Comment:
if its enabled by default, then we may need to merge https://github.com/ceph/s3-tests/pull/460/files .. I can update the PR with only the subset of tests which are currently failing.